### ISSUE TITLE
path的fill使用url方式，会导致fill-rule失效

### ISSUE THREAD

**Issue by xiaolieask** [react-native-oh-library/react-native-harmony-svg#202]
svg数据如下：
```
import React, { Component } from 'react'
import {
    StyleSheet,
    View,
    Text,
    ScrollView,
    Image,
    TouchableHighlight
} from 'react-native'

import { Path, Svg, Circle, Defs, SvgXml, Use, SvgCss } from 'react-native-svg'

const icon_m_发票管理 = `<svg width="240" height="240" viewBox="0 0 240 240" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M16 36C16 27.1634 23.1634 20 32 20H208C216.837 20 224 27.1634 224 36V68C224 76.8366 216.837 84 208 84V192.221C208 208.661 189.267 218.08 176.071 208.274L172.934 205.943C167.564 201.952 160.278 201.725 154.669 205.371L128.722 222.242C123.418 225.69 116.582 225.69 111.278 222.242L85.3311 205.371C79.7222 201.725 72.4364 201.952 67.0664 205.943L63.9288 208.274C50.7331 218.08 32 208.661 32 192.221L32 84C23.1634 84 16 76.8366 16 68V36ZM52 192.221V64H188V192.221L184.862 189.89C172.78 180.911 156.387 180.398 143.767 188.604L120 204.057L96.2332 188.604C83.6131 180.398 67.2201 180.911 55.1376 189.89L52 192.221ZM176 98C176 103.523 171.523 108 166 108L78 108C72.4771 108 68 103.523 68 98C68 92.4771 72.4771 88 78 88L166 88C171.523 88 176 92.4772 176 98ZM126 148C131.523 148 136 143.523 136 138C136 132.477 131.523 128 126 128L78 128C72.4772 128 68 132.477 68 138C68 143.523 72.4772 148 78 148L126 148Z" fill="url(#paint0_linear_14651_25058)"/><defs><linearGradient id="paint0_linear_14651_25058" x1="120" y1="224.828" x2="37.6064" y2="61.5868" gradientUnits="userSpaceOnUse"><stop stop-color="#FE3A3C"/><stop offset="1" stop-color="#FF5D5D"/></linearGradient></defs></svg>`

const icon_m_亲情卡 = `<svg width="240" height="240" viewBox="0 0 240 240" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd"
    d="M12 56V184C12 201.673 26.3269 216 44 216H196C213.673 216 228 201.673 228 184V56C228 38.3269 213.673 24 196 24H44C26.3269 24 12 38.3269 12 56ZM32 60V180C32 188.837 39.1634 196 48 196H192C200.837 196 208 188.837 208 180V60C208 51.1634 200.837 44 192 44H48C39.1634 44 32 51.1634 32 60Z"
    fill="url(#paint0_linear_14651_25076)" />
<path d="M220 104L20 104L20 64L220 64L220 104Z" fill="#FF5354" />
<path fill-rule="evenodd" clip-rule="evenodd"
    d="M177.613 152.003C177.613 137.859 166.147 126.392 152.003 126.392L134.421 126.392C121.151 126.392 110.393 137.15 110.393 150.421V152.003C110.393 157.311 114.696 161.614 120.004 161.614C125.312 161.614 129.615 157.311 129.615 152.003V150.421C129.615 147.766 131.767 145.615 134.421 145.615L152.003 145.615C155.531 145.615 158.391 148.475 158.391 152.003C158.391 155.531 155.531 158.391 152.003 158.391H145.215C139.907 158.391 135.604 162.694 135.604 168.002C135.604 173.31 139.907 177.613 145.215 177.613H152.003C166.147 177.613 177.613 166.147 177.613 152.003Z"
    fill="#FF3D3F" />
<path fill-rule="evenodd" clip-rule="evenodd"
    d="M62.3906 152.001C62.3906 166.145 73.8568 177.611 88.0011 177.611H105.583C118.853 177.611 129.611 166.854 129.611 153.583V152.001C129.611 146.693 125.308 142.39 120 142.39C114.691 142.39 110.388 146.693 110.388 152.001V153.583C110.388 156.237 108.237 158.389 105.583 158.389L88.0011 158.389C84.4731 158.389 81.6131 155.529 81.6131 152.001C81.6131 148.473 84.4731 145.613 88.0011 145.613H94.7887C100.097 145.613 104.4 141.31 104.4 136.002C104.4 130.694 100.097 126.391 94.7887 126.391L88.0011 126.391C73.8568 126.391 62.3906 137.857 62.3906 152.001Z"
    fill="#FF3D3F" />
<defs>
    <linearGradient id="paint0_linear_14651_25076" x1="120" y1="32.3478" x2="120" y2="209.739"
        gradientUnits="userSpaceOnUse">
        <stop offset="0.0131229" stop-color="#FF5D5D" />
        <stop offset="0.976103" stop-color="#FE3A3C" />
    </linearGradient>
</defs>
</svg>`

const icon_m_同城跑腿 = `<svg width="240" height="240" viewBox="0 0 240 240" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd"
    d="M199.544 187.454L139.544 221.054C127.402 227.854 112.598 227.854 100.456 221.054L40.4559 187.454C27.8236 180.38 20 167.032 20 152.554V87.4436C20 72.9654 27.8236 59.6174 40.4559 52.5434L100.456 18.9434C112.598 12.1437 127.402 12.144 139.545 18.9437L152 25.9186L190 47.1986L199.544 52.5432C212.176 59.6173 220 72.9654 220 87.4436V152.554C220 167.032 212.176 180.38 199.544 187.454ZM193.783 167.418L125.783 204.818C122.182 206.799 117.818 206.799 114.217 204.818L52.4339 170.837C44.7648 166.619 40 158.561 40 149.808V83.0939C40 78.7176 42.3824 74.6883 46.217 72.5793L114.217 35.1793C117.818 33.1987 122.182 33.1988 125.783 35.1793L193.783 72.5793C197.618 74.6883 200 78.7176 200 83.0939V156.903C200 161.28 197.618 165.309 193.783 167.418Z"
    fill="url(#paint0_linear_14651_25085)" />
<path
    d="M126.549 108.851C122.848 110.995 118.285 111.006 114.572 108.88L99.9756 100.521C94.6424 97.4669 88 101.317 88 107.463V142.076C88 144.935 89.5261 147.577 92.0029 149.005L115.662 162.652C118.785 164.449 122.354 164.449 125.477 162.652L204 117.258V64L126.549 108.851Z"
    fill="#FF4142" />
<defs>
    <linearGradient id="paint0_linear_14651_25085" fill="white" x1="151.818" y1="13.8438" x2="120.385" y2="226.211"
        gradientUnits="userSpaceOnUse">
        <stop stop-color="#FE3A3C" fill="white" />
        <stop offset="1" stop-color="#FF5D5D" fill="white" />
    </linearGradient>
</defs>
</svg>`
export default class VideoCell extends Component {
    render = () => {
        return (
            <ScrollView>
                <SvgCss
                    xml={icon_m_发票管理}
                    width={40}
                    height={40}
                    fill={"black"}
                    style={{ marginLeft: 20, marginTop: 20 }}
                />

                <SvgCss
                    xml={icon_m_亲情卡}
                    width={40}
                    height={40}
                    fill={"black"}
                    style={{ marginLeft: 20, marginTop: 20 }}
                />

                <SvgCss
                    xml={icon_m_同城跑腿}
                    width={40}
                    height={40}
                    fill={"black"}
                    style={{ marginLeft: 20, marginTop: 20 }}
                />
            </ScrollView>
        )
    }
}
```

正确效果图：
![image](https://github.com/react-native-oh-library/react-native-harmony-svg/assets/26913643/c400ef70-9b64-47ce-81fc-85c421433e14)
鸿蒙显示效果图：
![image](https://github.com/react-native-oh-library/react-native-harmony-svg/assets/26913643/000183a2-a077-442b-99b1-973f413fadcf)

**Comment by Louis-C7** [comment]
solve in [PR#199](https://github.com/react-native-oh-library/react-native-harmony-svg/pull/199)

**Comment by Louis-C7** [comment]
Solved in [PR#199](https://github.com/react-native-oh-library/react-native-harmony-svg/pull/199)
